_topic_map.yml

```diff
@@ -114,6 +114,8 @@ Topics:
     File: init-bundle-ocp
   - Name: Installing secured cluster services for RHACS on Red Hat OpenShift
     File: install-secured-cluster-ocp
+  - Name: Optional - Configuring Secured cluster configuration options for RHACS using the Operator
```

Comment:
Possibly out of scope, but should these "Optional -" (both in the topic_map and headings) be "Optional: "? While RHACS looks to be the first to actually have it in a topic_map (I see there is also another introduced by an earlier PR), in other openshift-docs branches it looks like we've followed the [IBM Style Guide's guidance on optional steps](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#indicating-optional-and-conditional-steps) (which uses colons) for headings, at least.

Reply:
Thank you @adellape for the suggestion, I'll get it done as a follow-up PR.